src/main/java/de/neuland/jade4j/parser/node/Node.java

```diff
@@ -16,9 +18,10 @@
     protected String value;
     protected Node block;
     protected String fileName;
+    protected Map<String, String> translation = Collections.emptyMap();
```

Comment:
I think it would be good to rename this so that's it's clearer what this map is. e.g. `translationsByLanguageCode`

same nit, I think should be `translations`.